pyomo/contrib/mindtpy/iterate.py

```diff
@@ -307,7 +307,7 @@ def algorithm_should_terminate(solve_data, config, check_cycling):
             solve_data.curr_int_sol = get_integer_solution(solve_data.mip)
             if config.cycling_check and solve_data.mip_iter >= 1:
                 if solve_data.curr_int_sol in set(solve_data.integer_list):
-                    config.logger.warning(
+                    config.logger.info(
```

Comment:
Why changing this from warning to info?

Reply:
The warning makes the output of the test really nasty.